2022-12-15 03:37:10.207 +02:00 [ERR] Error: {"ClassName":"System.Exception","Message":"TaskResult","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SpendLess.UnitTests.Tests.<>c.<<TaskResultMethodsWithExceptionsAreLoggedByInterceptor>b__32_0>d.MoveNext() in D:\\C#\\zapas\\master\\SpendLess\\UnitTests\\Tests.cs:line 255\r\n--- End of stack trace from previous location ---\r\n   at SpendLess.Server.Interceptor.UnhandledExceptionLogger.InterceptAsync[T](Task`1 task) in D:\\C#\\zapas\\master\\SpendLess\\Server\\Interceptor\\UnhandledExceptionLogger.cs:line 63","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"Tests","WatsonBuckets":null}
2022-12-15 03:37:10.293 +02:00 [ERR] Error happened in method: . Error: {"ClassName":"System.Exception","Message":null,"Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at Moq.Behaviors.ThrowException.Execute(Invocation invocation) in C:\\projects\\moq4\\src\\Moq\\Behaviors\\ThrowException.cs:line 22\r\n   at Moq.MethodCall.ExecuteCore(Invocation invocation) in C:\\projects\\moq4\\src\\Moq\\MethodCall.cs:line 110\r\n   at Moq.Setup.Execute(Invocation invocation) in C:\\projects\\moq4\\src\\Moq\\Setup.cs:line 84\r\n   at Moq.FindAndExecuteMatchingSetup.Handle(Invocation invocation, Mock mock) in C:\\projects\\moq4\\src\\Moq\\Interception\\InterceptionAspects.cs:line 95\r\n   at Moq.Mock.Moq.IInterceptor.Intercept(Invocation invocation) in C:\\projects\\moq4\\src\\Moq\\Interception\\Mock.cs:line 17\r\n   at Moq.CastleProxyFactory.Interceptor.Intercept(IInvocation underlying) in C:\\projects\\moq4\\src\\Moq\\Interception\\CastleProxyFactory.cs:line 107\r\n   at Castle.DynamicProxy.AbstractInvocation.Proceed()\r\n   at Castle.Proxies.IInvocationProxy.Proceed()\r\n   at SpendLess.Server.Interceptor.UnhandledExceptionLogger.Intercept(IInvocation invocation) in D:\\C#\\zapas\\master\\SpendLess\\Server\\Interceptor\\UnhandledExceptionLogger.cs:line 29","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"Moq","WatsonBuckets":null}
2022-12-15 03:37:21.753 +02:00 [ERR] CLIENT SIDE ERROR
User's email: email
Token exp.: date
Message:Exception of type 'System.Exception' was thrown.
Stack trace: 
